csv_detective/detect_fields/other/json/__init__.py

```diff
@@ -0,0 +1,13 @@
+import json
+
+PROPORTION = 1
+
+
+def _is(val):
+    '''Detects json'''
+    try:
+        loaded = json.loads(val)
+        if isinstance(loaded, dict) and not(any([geo in loaded for geo in ['coordinates', 'geometry']])):
```

Comment:
What about lists?

Reply:
I might be wrong but I don't think anything else than an object (dict) or an array (list) can be expressed.

Reply:
Changed ✅